sdk/daml-script/test/src/main/scala/com/digitalasset/daml/lf/engine/script/Daml2ScriptTestRunner.scala

```diff
@@ -43,8 +43,8 @@ class Daml2ScriptTestRunner extends DamlScriptTestRunner {
           |ExceptionSemantics:uncaughtUserException SUCCESS
           |ExceptionSemantics:unhandledArithmeticError FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submit failed: FAILED_PRECONDITION: UNHANDLED_EXCEPTION(9,XXXXXXXX): Interpretation error: Error: Unhandled Daml exception: DA.Exception.ArithmeticError:ArithmeticError@XXXXXXXX{ message = "ArithmeticError while evaluating (DIV_INT64 1 0)." }. Details: Last location: [DA.Internal.Template.Functions:265], partial transaction: ...
           |ExceptionSemantics:unhandledUserException FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submit failed: FAILED_PRECONDITION: UNHANDLED_EXCEPTION(9,XXXXXXXX): Interpretation error: Error: Unhandled Daml exception: ExceptionSemantics:E@XXXXXXXX{  }. Details: Last location: [DA.Internal.Exception:176], partial transaction: ...
-          |ExceptionSemanticsWithKeys:duplicateKey FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submitMustFail failed: null
-          |LFContractKeys:lookupTest FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submitMustFail failed: null
+          |ExceptionSemanticsWithKeys:duplicateKey FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submitMustFail failed: SErrorDamlException(UserError(Expected submit to fail but it succeeded)
+          |LFContractKeys:lookupTest FAILURE (com.digitalasset.daml.lf.engine.script.Script$FailedCmd: Command submitMustFail failed: SErrorDamlException(UserError(Expected submit to fail but it succeeded)
```

Comment:
I don't understand why this (and the equivalent lines in `Daml3ScriptTestRunner.scala`) changed, but a `submitMustFail` _not_ failing seems like a bug